### ISSUE TITLE
Verify Python is cross version compatible

### ISSUE THREAD

**Issue by Myoldmopar** [NREL/EnergyPlus#4888]
The python scripts added for automated documentation builds were only tested on Python 2.7, and most likely need a few tweaks to get working on Python 3.  Check this with any other Python in the project.  It will make @mbadams5 happy.  :smile:

**Comment by Myoldmopar** [comment]
Here's a full list of the Python files in the repo, with a number of subdirectories ignored:

```
$ find . -name '*.py' | grep -v '^./third_party\|^./cmake-build-debug\|^./builds\|^./bin\|^./workflows' 
./doc/tools/example_file_objects.py
./doc/tools/make_pdfs.py
./doc/tools/idd_parser_library.py
./doc/tools/idf_summary_class.py
./doc/tools/idf_parser_library.py
./doc/tools/make_html.py
./doc/tools/parse_output_variables.py
./doc/tools/example_file_summary.py
./doc/tools/create_changelog.py
./doc/tools/parse_ems_actuators.py
./scripts/dev/check_non_utf8.py
./scripts/dev/make_pretty.py
./scripts/dev/license-check.py
./scripts/dev/clear_coverage_results.py
./scripts/dev/179D_Generate_Index.py
./scripts/dev/licensetext.py
./scripts/dev/generate_epJSON_schema/generate_epJSON_schema.py
./scripts/dev/generate_epJSON_schema/modify_schema.py
./scripts/dev/generate_epJSON_schema/idd_parser.py
./scripts/dev/verify_idfs_in_cmake.py
./scripts/dev/transition_run_check.py
./scripts/dev/validate_idd_units.py
./scripts/dev/check_stray_fields_in_idd.py
./scripts/dev/verify_file_encodings.py
./scripts/dev/check_pr_labels.py
./scripts/dev/change_version.py
./scripts/dev/license-update.py
./src/Transition/transition_coil_heating_gas.py
```

I'll try to address these all in a PR.

**Comment by Myoldmopar** [comment]
Fixed via #7309
